docs/tutorials/portal/103/portal-103-4.rst

```diff
@@ -69,10 +70,13 @@ Spatial constraints are applied to the ``FROM`` table, not the ``JOIN`` table.
         AND cv.expMidptMJD > 60631 AND cv.expMidptMJD < 60637
         AND src.band = 'i'
 
+If a lightcurve appears in the left panel instead of a coverage HIPS map as shown in Figure 1, click the tab that says “Coverage” to display the HIPS map.
+
 **4. Review the two-table join results.**
 Notice that this join is not one-to-one: there are multiple individual sources returned that are matched to the same visit.
 In other words, there are multiple rows from the ``Source`` table joined with a given row from the ``CcdVisit`` table.
-If multiple tabs are present above the upper left panel in the default Results tab layout, click the "Coverage" tab to display the coverage chart.
+
+.. If multiple tabs are present above the upper left panel in the default Results tab layout, click the "Coverage" tab to display the coverage chart.
```

Comment:
Why was this commented out? It is basically giving the instruction this ticket was seeking to add, in cases when it wasn't already there.